kratos/python_scripts/modelers/import_model_part_modeler.py

```diff
@@ -0,0 +1,50 @@
+import KratosMultiphysics
+
+class ImportModelPartModeler(KratosMultiphysics.Modeler):
+
+    def __init__(self, model, settings):
+        super().__init__(model, settings)
+
+        # Cannot validate as settings may differ ammong input types
+        settings.AddMissingParameters(self.__GetDefaultSettings())
+
+        # Declare required member variables
+        self.model_part = None
+        self.settings = settings
+
+        # Create the import destination model part
+        # It is mandatory to do this when the modeler is instantiated to have the model part created before the solvers add the variables
+        model_part_name = self.settings["model_part_name"].GetString()
+        if not model_part_name:
+            err_msg = "Missing 'model_part_name' in input settings. This is where the imported model part is to be stored."
+            raise Exception(err_msg)
+        else:
+            self.model_part = model.CreateModelPart(model_part_name)
+
+    def SetupGeometryModel(self):
+        super().SetupGeometryModel()
+
+    def PrepareGeometryModel(self):
+        super().PrepareGeometryModel()
+
+    def SetupModelPart(self):
+        super().SetupModelPart()
+
+        # Import the model part data
+        # Note that at this point solvers must have added the variables to the nodal variable data
+        KratosMultiphysics.SingleImportModelPart.Import(
+            self.model_part,
+            self.settings,
+            self.settings["input_type"].GetString())
+
+    def __GetDefaultSettings(self):
+        default_settings = KratosMultiphysics.Parameters('''{
+            "echo_level" : 0,
+            "input_type" : "mdpa",
+            "input_filename" : "",
```

Comment:
It would be great if we put the extension in the filename to avoid one of our historical problems.

Reply:
Resolved below.